### PR TITLE
Issue 255 new repo

### LINKED ISSUE #255: Content Type: Script Page
Alongside converting this to D8 we need to future proof this to work with 

- [ ] agilepoint [replacing seamlessdocs as a service]

- [ ] google forms

### PR DESCRIPTION
**Pull request by jimafisk**
Fixes #255

### CODE REVIEW

File: docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{#
+ * @file
+ * Returns the HTML for a script page.
+ *
+ * Complete documentation for this file is available online.
+ *
+ * @see https://drupal.org/node/1728164
+ #}
+
+<article id="node-{{ node.id }}" {{ attributes }}>
```
Comment (davidrkupton) [inline docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig]: you can add 
```$variables['attributes']->setAttribute('id','node-' . $variables['content']['#node']->id())``` 
to the `node_script_page_preprocess_node__script_page()`.  That way you can simplify to 
```<article{{ attributes }}>``` 
here and all attributes are set in the preprocess.

File: docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          <div class="list-item-wrapper">
+            {{ content.field_sidebar_components }}
+          </div>
+        {% endif %}
+      </div>
+    </div>
+    <div class="clearfix"></div>
+  </div>
+  {% if not node.field_components.isempty %}
+    <div class="department-components desktop-100" {{ content_attributes }}>
```
Comment (davidrkupton) [inline docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig]: Similarly, the classes listed can be added to the `content_attributes` array in the preprocess hook.

File: docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ * Complete documentation for this file is available online.
+ *
+ * @see https://drupal.org/node/1728164
+ #}
+
+<article id="node-{{ node.id }}" {{ attributes }}>
+  <div class="department-info-wrapper desktop-100">
+    <div class="column mobile-100 desktop-66-left">
+      <h1 class="title">{{ title }}</h1>
```
Comment (davidrkupton) [inline docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig]: I am having to use `title.0` to get this to print out ...

Comment (jimafisk) [reply docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig]: I've noticed similar inconsistencies, not sure what the "rule" is. Somehow title seems to be working on this node:

title

Sometimes `node.label` works too.

File: docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig (hunk 2 above)
Comment (jimafisk) [reply docroot/modules/custom/bos_content/modules/node_script_page/templates/node--script-page.html.twig]: That makes sense, I can update those tomorrow. Thanks!

**Pull request merged**